packages/core/src/api-wrappers/createApp.tsx

```diff
@@ -97,16 +97,32 @@ export function createApp(options?: AppOptions) {
     <ErrorPage status="404" statusMessage="PAGE NOT FOUND" />
   );
   const DefaultBootErrorPage = ({ step, error }: BootErrorPageProps) => {
-    let message = '';
-    if (step === 'load-config') {
-      message = `The configuration failed to load, someone should have a look at this error: ${error.message}`;
+    switch (step) {
```

Comment:
Could keep the existing message assignment right? Or does it need to not be in a router? In that case we got a bit of a problem too, maybe it's possible to detect if we're in one or not x)

Reply:
Yep, this is much better.

Thanks to @dhenneke for pointing me to the right hook!